components/permit-holders/MedicalHistoryCard.tsx

```diff
@@ -0,0 +1,69 @@
+import { Box, Text, Divider, Link } from '@chakra-ui/react'; // Chakra UI
+import Table from '@components/internal/Table'; // Table component
+import PermitHolderInfoCard from '@components/internal/PermitHolderInfoCard';
+import { Aid, MedicalInformation } from '@lib/graphql/types'; // Aid enum & MedicalInformation type
+import MedicalHistoryModal from '@components/permit-holders/modals/MedicalHistoryModal'; // Medical History Modal
+
+// Placeholder data
+const mockMedicalHistory = {
+  disability: 'Condition 1',
+  affectsMobility: true,
+  mobilityAidRequired: true,
+  cannotWalk100m: true,
+  aid: [Aid.Cane, Aid.ElectricChair],
+  notes: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+  certificationDate: '2021/01/01',
+};
+
+const DATA = Array(4).fill({
+  disablingCondition: 'Condition 1',
+  associatedApp: { associatedApp: 12345 },
+  dateUploaded: '2021/01/01',
+  fileUrl: { fileUrl: '/' },
+  application: { application: mockMedicalHistory },
+});
+
+const COLUMNS = [
+  {
+    Header: 'Disabling Condition(s)',
+    accessor: 'disablingCondition',
+    disableSortBy: true,
+    minWidth: 250,
+  },
+  {
+    Header: 'Certification Date',
+    accessor: 'dateUploaded',
+    disableSortBy: true,
+    // NOTE: Check value
```

Comment:
If this is a TODO for future dev work then I think you should add more context. Otherwise if you could resolve it that would be great

Reply:
Just a note to myself that I forgot to remove, nice catch!